src/containers/login/index.js

```diff
@@ -27,12 +27,12 @@ const Login = props => (
                         </div>
                         <TextField placeholder="Password" type="password" class="field" /><br/>
                         <div class="btnContainer">
-                            <button type="button" onClick={() => props.changeToBody()} class="btn">LOGIN</button> 
+                            <button type="button" onClick={() => props.changeToBody()} class="btn">LOGIN</button>
                         </div>
                     </MuiThemeProvider>
-                </form> 
+                </form>
             </div>
-            
+
         </div>
         <div class="signUpBackground">
             <p class="btnGrey" onClick={() => props.changeToRegister()} class="signUpLink" >No account yet? Get setup now</p>

```

Comment:
Just so you know: You don't require the lambda notation `() =>` if you're not passing in parameters